app/models/sail/setting.rb

```diff
@@ -33,7 +33,7 @@ class UnexpectedCastType < StandardError; end
     after_initialize :instantiate_caster
 
     scope :paginated, lambda { |page, per_page|
-      select(:name, :description, :group, :value, :cast_type, :updated_at)
+      select(:id, :name, :description, :group, :value, :cast_type, :updated_at)
```

Comment:
Can we just remove the select all together? I think it makes sense at this point since we're basically selecting all the fields.